jenkins_pipelines/environments/uyuni-master-dev-acceptance-tests-sonarqube

```diff
@@ -0,0 +1,141 @@
+                }
+            }
+
+            stage('Deploy') {
+                // Provision the environment
+                if (params.terraform_init) {
+                    env.TERRAFORM_INIT = '--init'
+                } else {
+                    env.TERRAFORM_INIT = ''
+                }
+                env.TERRAFORM_TAINT = ''
+                if (params.terraform_taint) {
+                    switch(params.sumaform_backend) {
+                        case "libvirt":
+                            env.TERRAFORM_TAINT = " --taint '.*(domain|main_disk).*'";
+                            break;
+                        case "aws":
+                            env.TERRAFORM_TAINT = " --taint '.*(host).*'";
+                            break;
+                        default:
+                            println("ERROR: Unknown backend ${params.sumaform_backend}");
+                            sh "exit 1";
+                            break;
+                    }
+                }
+                sh "set +x; source /home/jenkins/.credentials set -x; export TF_VAR_CUCUMBER_GITREPO=${params.cucumber_gitrepo}; export TF_VAR_CUCUMBER_BRANCH=${params.cucumber_ref}; export TERRAFORM=${params.terraform_bin}; export TERRAFORM_PLUGINS=${params.terraform_bin_plugins}; ./terracumber-cli ${common_params} --logfile ${resultdirbuild}/sumaform.log ${env.TERRAFORM_INIT} ${env.TERRAFORM_TAINT} --sumaform-backend ${params.sumaform_backend} --runstep provision"
+                deployed = true
+            }
+
+            stage('Enable JaCoCo Agent') {
+                def url = "https://search.maven.org/remotecontent?filepath=org/jacoco/org.jacoco.agent/0.8.7/org.jacoco.agent-0.8.7-runtime.jar"
+                sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'ssh $SERVER \'wget ${url} -O /tmp/jacocoagent.jar\' "
+                def regex = "sed -i 's/.$/ -javaagent:\/tmp\/jacocoagent.jar=output=tcpserver,address=*,port=6300" /' /etc/sysconfig/tomcat"
+                sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'ssh $SERVER \'${regex}\' "
+                sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'ssh $SERVER \'systemctl restart tomcat.service\' "
+            }
+
+            stage('Sanity Check') {
+                sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:sanity_check'"
+            }
+
+            stage('Core - Setup') {
+                sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:core'"
+            }
```

Comment:
I was wondering if we should skip that step for the testing right now to speed up the pipeline?